utp-testing/src/main.rs

```diff
@@ -43,20 +42,13 @@ impl TestApp {
     }
 
     async fn process_utp_request(&self) {
-        let mut event_stream = self
-            .utp_listener
-            .read()
-            .await
-            .discovery
-            .discv5
-            .event_stream()
-            .await
-            .unwrap();
+        let mut event_stream = self.discovery.discv5.event_stream().await.unwrap();
 
-        let listener = Arc::clone(&self.utp_listener);
+        let utp_sender = self.utp_event_tx.clone();
 
         tokio::spawn(async move {
             while let Some(event) = event_stream.recv().await {
+                debug!("event: {event:?}");
```

Comment:
Probably worth adding a bit more context into this log, like:
```suggestion
                debug!("utp-testing TestApp handling event: {event:?}");
```